proto/cacheclient.proto

```diff
@@ -70,6 +72,29 @@ message _SetResponse {
   string message = 2;
 }
 
+message _KeyValuePair {
+  bytes field = 1;
+  bytes value = 2;
+}
+
+message _SetMultipleIfNotExistsRequest {
+  repeated _KeyValuePair items = 1;
+  uint64 ttl_milliseconds = 2;
+  bool refresh_ttl = 3;
+}
+
+message _SetMultipleIfNotExistsResponse {
+  message _SetMultipleIfNotExistsResponsePart {
+    oneof result {
+      Ok ok = 1;
+      NotStored not_stored = 2;
```

Comment:
Hm... wondering if these should be `_Ok` and `_NotStored`, based on https://github.com/momentohq/client-protos/blob/04cd7201a2f8bbba3c2011a1f509ca7373538699/proto/cacheclient.proto#L115-L118